README.md

```diff
@@ -25,9 +25,9 @@ Before the build all dependencies must be downloaded and laid out as follows:
 This can be done using the following commands:
 ```bash
 cd <workspace>
-git clone https://github.com/llvm/llvm-project.git .
-git clone https://github.com/KhronosGroup/SPIRV-LLVM-Translator.git
-git clone https://github.com/intel/opencl-clang.git
+git clone https://github.com/llvm/llvm-project.git . -b release/11.x
+git clone https://github.com/KhronosGroup/SPIRV-LLVM-Translator.git -b llvm_release_110
+git clone https://github.com/intel/opencl-clang.git -b ocl-open-110
```

Comment:
why checkout latest trunk and rename to a branch with specific version?
Suggest to keep it as is?

Reply:
This is a new branch, and we modified it based on master?

Reply:
Yes. This is new branch. It's not renamed to branch. See git help below:
       --branch <name>, -b <name>
           Instead of pointing the newly created HEAD to the branch pointed to by the cloned repository's HEAD, point to <name> branch instead. In a non-bare
           repository, this is the branch that will be checked out.  --branch can also take tags and detaches the HEAD at that commit in the resulting repository.

Reply:
I see, but I think in readme, we should checkout master for illustration.

Reply:
This readme.md includes the instructions for ocl-open-110 branch, and is showed when ocl-open-110 branch is selected. There is another readme.md in master branch.

Reply:
sorry I missed that this is for ocl-open-110 branch.